src/Raven.Studio/typescript/viewmodels/resources/widgets/databaseOverviewWidget.ts

```diff
@@ -0,0 +1,95 @@
+import appUrl = require("common/appUrl");
+import perNodeStatItems = require("models/resources/widgets/perNodeStatItems");
+import databaseOverviewItem = require("models/resources/widgets/databaseOverviewItem");
+
+class databaseOverviewWidget extends abstractDatabaseAndNodeAwareTableWidget<Raven.Server.Dashboard.Cluster.Notifications.DatabaseOverviewPayload,
+    perNodeStatItems<databaseOverviewItem>, databaseOverviewItem> {
+    
+    getType(): Raven.Server.Dashboard.Cluster.ClusterDashboardNotificationType {
+        return "DatabaseOverview";
+    }
+
+    constructor(controller: clusterDashboard) {
+        super(controller);
+
+        for (const node of this.controller.nodes()) {
+            const stats = new perNodeStatItems<databaseOverviewItem>(node.tag());
+            this.nodeStats.push(stats);
+        }
+    }
+
+    protected createNoDataItem(nodeTag: string, databaseName: string): databaseOverviewItem {
+        return databaseOverviewItem.noData(nodeTag, databaseName);
+    }
+
+    protected mapItems(nodeTag: string, data: Raven.Server.Dashboard.Cluster.Notifications.DatabaseOverviewPayload): databaseOverviewItem[] {
+        return data.Items.map(x => new databaseOverviewItem(nodeTag, x));
+    }
+    
+    protected manageItems(items: databaseOverviewItem[]): void {
+        if (items.length) {
+            const firstItem = items[0];
+            let firstDbName = firstItem.database;
+            let commonItem = databaseOverviewItem.commonData(firstItem.nodeTag, firstItem.database, firstItem.documents, firstItem.indexes, firstItem.ongoingTasks, firstItem.backupInfo);
+            items.unshift(commonItem);
+            
+            if (items.length > 2) {
+                for (let i = 2; i < items.length - 1; i++) {
+                    const item = items[i];
+                    let dbName = item.database;
+                    if (dbName != firstDbName) {
+                        commonItem = databaseOverviewItem.commonData(item.nodeTag, item.database, item.documents, item.indexes, item.ongoingTasks, item.backupInfo);
+                        items.splice(i, 0, commonItem);
+                        firstDbName = dbName;
+                    }
+                }
+            }
+        }
+    }
+
+    protected applyPerDatabaseStripes(items: databaseOverviewItem[]) {
+        for (let i = 0; i < items.length; i++) {
+            const item = items[i];
+            
+            if (item.isCommonItem) {
+                item.even = true;
+            } else {
+                item.even = false;
+                item.hideDatabaseName = true;
+            }
+        }
+    }
+
+    protected prepareColumns(containerWidth: number, results: pagedResult<databaseOverviewItem>): virtualColumn[] {
+        const grid = this.gridController();
+        return [
+            new textColumn<databaseOverviewItem>(grid, x => x.hideDatabaseName ? "" : x.database, "Database", "20%"),
```

Comment:
I wonder if we need common item concept. Can't we just relay to hideDatabaseName?

Reply:
Imho - I think we need the concept of `commonItem ` on the items
otherwise - you end up with code like this (on nodeTagColumn.ts)

```
        if (item.hideDatabaseName) {
            return `<span class="node-label ${extraClass}" title="Go to ${this.hrefProvider(item).targetDescription} 
        } else {
            return "";
        }
```

but you don't want this code 
since we have panels where the node tag shows **both** in the common item **and** below, in a separate item